content/en/docs/tasks/run-application/mysql-deployment.yaml

```diff
@@ -9,11 +9,27 @@ spec:
     app: mysql
   clusterIP: None
 ---
+kind: PersistentVolume
```

Comment:
Add a new resource to the YAML is fine, but you have to change the tests/example_test.go file to change the test case as well. Travis won't feel happy about it unless the tests pass.